src/www/widgets/widgets/gateways.widget.php

```diff
@@ -78,6 +80,7 @@ function gateways_widget_update(sender, data)
         <tr>
             <th><?=gettext('Name')?></th>
             <th><?=gettext('RTT')?></th>
+            <th><?=gettext('Dev')?></th>
```

Comment:
Let's use STDEV consistently... https://www.abbreviations.com/abbreviation/Standard+Deviation

Reply:
Still think it should be RTTd :)